zschema/compounds.py

```diff
@@ -84,27 +84,29 @@ def docs_es(self, parent_category=None):
             retv["doc"] = self.doc
         return retv
 
-    def validate(self, name, value, policy=_NO_ARG, parent_policy=_NO_ARG):
+    def validate(self, name, value, policy=_NO_ARG, parent_policy=_NO_ARG, path=_NO_ARG):
         calculated_policy = self._calculate_policy(name, policy, parent_policy)
+        if not path:
```

Comment:
Double checking: `_NO_ARG` is false-y because it has 

```python
class _NO_ARG(object):
    __nonzero__ = lambda _: False
```

Reply:
yep, I had to confirm that myself, too.